.travis.yml

```diff
@@ -1,7 +1,9 @@
 language: android
 jdk: oraclejdk8
 sudo: required
-dist: precise # For 7.5GB of memory since the emulator requires a big chunk.
+# Use trusty distribution to resolve the adb(requires at least GLIBC_2.16 version) execution failure
+# Reference: https://github.com/atom/atom/issues/14440#issuecomment-302158430
+dist: trusty # For 7.5GB of memory since the emulator requires a big chunk.
```

Comment:
Would be great to add this link here too:
https://docs.travis-ci.com/user/reference/trusty/